### DISCUSSION TITLE
Internal server error when download filename contains non-ascii character

### DISCUSSION THREAD

**Discussion by NateScarlet** [requarks/wiki#4831] [Error / Bug Report]
```
wiki_1      | 2021-12-27T07:46:27.658Z [MASTER] error: Invalid character in header content ["Content-disposition"]
wiki_1      | 2021-12-27T07:47:38.404Z [MASTER] error: Invalid character in header content ["Content-disposition"]
wiki_1      | 2021-12-27T07:47:45.072Z [MASTER] error: Invalid character in header content ["Content-disposition"]
```

## repoduction

1. upload a file named as `测试.txt`
2. download that file

## workaround

disable `Force Download of Unsafe Extensions` from /a/security

## posibble solution 

encode filename here

see https://tools.ietf.org/html/rfc2231

https://github.com/Requarks/wiki/blob/e84c15b926843a32d23b7e71e2d2f4622dfca9e7/server/models/assets.js#L177

**Comment by pwjpj52** [comment]
同样问题

**Comment by pwjpj52** [comment]
i have the same problem
